src/URLHelper.php

```diff
@@ -5,6 +5,48 @@
 use Exception;
 
 class URLHelper {
+    /*
+     * Returns the current full URL including querystring
+     *
+     * @return string
+     */
+    public static function getCurrent() : string {
+        $scheme = $_SERVER['SERVER_PORT'] == 80 ? 'http' : 'https';
```

Comment:
Needs to cover custom ports, many localhost setups or such will be on non-standard web ports

Reply:
Fixed through https://github.com/WP2Static/wp2static/pull/617/commits/3246e34af933811fe506d4f563bac9bab27188eb